index.js

```diff
@@ -33,17 +33,16 @@ exports.handler = function (event, context, callback) {
   }
 
   api_request(params, function (err, response) { // get packages from NE API
+    var body = JSON.parse(JSON.stringify(params));
+    body.items = []; // send an empty array to the client so it knows wazzup!
+    body.searchComplete = true;
+    AwsHelper.pushResultToClient(body, () => {
+      AwsHelper.log.info({ err: err, packages: response.result.length },
+        `Package search complete: ${response.result.length} packages found`);
```

Comment:
Seeing we have the `packages` object in the payload, adding it in the message adds no real benifit.